ports/nrf/main.c

```diff
@@ -230,11 +230,9 @@ pin_init0();
 
 led_state(1, 0);
 
-#if MICROPY_VFS || MICROPY_MBFS
```

Comment:
How about just changing this line to `#if MICROPY_VFS || MICROPY_MBFS || MICROPY_MODULE_FROZEN`?

Reply:
Done.